include/eigenpy/numpy-map.hpp

```diff
@@ -158,7 +158,7 @@ struct NumpyMapTraits<MatType, InputScalar, AlignmentValue, Stride, true> {
       rowMajor = (PyArray_DIMS(pyArray)[0] > PyArray_DIMS(pyArray)[1]) ? 0 : 1;
 
     assert((PyArray_DIMS(pyArray)[rowMajor] < INT_MAX) &&
-           (PyArray_STRIDE(pyArray, rowMajor)));
+           (PyArray_STRIDE(pyArray, rowMajor) < INT_MAX));
```

Comment:
According to https://numpy.org/doc/1.17/reference/c-api.array.html#c.PyArray_STRIDES, `PyArray_STRIDE`returns a pointer, not a value. What is then your issue exactly? Could you report the launch of the failing test?

Reply:
And thanks a for helping us to make EigenPy even stronger ;)

Reply:
The issue that it's `NULL` during the this test case:

https://github.com/stack-of-tasks/eigenpy/blob/e491c961c24e5860e21c63a596a7e8954598e51a/unittest/python/test_matrix.py#L13-L16

There's no message other than the assertion here. When PyArray_STRIDE is NULL, then the assertion will trigger.

Reply:
Then, we should remove rather remove this check.

Reply:
I'll defer to your best judgement, thanks for taking a look at all of this!